src/scripts/main.js

```diff
@@ -354,7 +354,42 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+function addRow(object) {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = object.name;
+  row.appendChild(nameCell);
+
+  const genderCell = document.createElement('td');
+
+  genderCell.textContent = object.sex;
+  row.appendChild(genderCell);
+
+  const bornCell = document.createElement('td');
+
+  bornCell.textContent = object.born;
+  row.appendChild(bornCell);
+
+  const diedCell = document.createElement('td');
+
+  diedCell.textContent = object.died;
+  row.appendChild(diedCell);
+
+  const ageCell = document.createElement('td');
+
+  ageCell.textContent = +object.died - +object.born;
+  row.appendChild(ageCell);
+
+  const centuryCell = document.createElement('td');
+
+  centuryCell.textContent = Math.ceil(object.died / 100);
+  row.appendChild(centuryCell);
+
+  table.appendChild(row);
+}
```

Comment:
This function is doing too much. It's creating the cells, adding the text, and appending the cells to the row. This makes the function harder to test and reuse. Consider breaking this function into smaller functions, each with a single responsibility. For example, you could have a function that creates a cell, another that adds text to a cell, and another that appends a cell to a row.